src/components/safe-apps/AppFrame/TransactionQueueBar/styles.module.css

```diff
@@ -3,7 +3,7 @@
   bottom: 0;
   right: 0;
   width: 100%;
-  z-index: 1;
+  z-index: 1300;
```

Comment:
Please check how this plays together with the onboard modals.

Reply:
I will use the `z-index: var(--onboard-modal-z-index, var(--modal-z-index));`   (`z-index: 1201;`) to make sure that the Transaction queue bar does not interfere with the onboard modals:

![Captura de pantalla 2022-11-16 a las 15 52 53](https://user-images.githubusercontent.com/26763673/202214763-fc6fdd3e-c742-4cd6-8adb-fcd1aa2b9813.png)